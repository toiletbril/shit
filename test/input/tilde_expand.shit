test -d ~ && echo 1
test -d ~/ && echo 2
test -d \~ && echo 3