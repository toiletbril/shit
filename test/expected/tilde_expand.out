[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "test -d ~"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 1"]
  [CompoundListCondition, None]
    [SimpleCommand "test -d ~/"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 2"]
  [CompoundListCondition, None]
    [SimpleCommand "test -d ~"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 3"]
1
2
[Code 1]
